MODULE.bazel

```
@@ -2,7 +2,18 @@
 # Exceptions. See /LICENSE for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-"""Bazel modules."""
+"""Bazel modules.
+
+If `MODULE.bazel.lock` changes locally, it means the host platform is not part
+of the GitHub supported set. Running `bazel mod deps` provides a canonical
+update to `MODULE.bazel.lock`; create a PR with those changes in order to
+include the host platform.
```

I don't really follow this -- this seems to imply more ties to GitHub than I would expect. For example, we don't have any GitHub support for any Arm platforms. And it would seem odd for users to send PRs expanding it.

Maybe something more like:
```suggestion
If `MODULE.bazel.lock` changes locally, it means the host platform hasn't yet been added to the lock file. Running `bazel mod deps` provides a canonical
update to `MODULE.bazel.lock`; create a PR with those changes in order to
include the host platform.
```